requirements.txt

```diff
@@ -39,7 +39,7 @@ rq==1.1.0
 rq-scheduler==0.9.1
 jsonschema==3.1.1
 RestrictedPython==5.0
-pysaml2==4.8.0
+pysaml2==6.1.0
```

Comment:
Are we sure this is backward compatible?

Reply:
yes.